sys/src/cmd/bc/bc.y

```diff
@@ -29,7 +29,7 @@
 	char*	pre;
 	char*	post;
 
-	long	peekc = -1;
+	int32_t	peekc = -1;
```

Comment:
This really ought to be an int if possible. Do we really have to say int32 in this case? It's just a char.